python/dolfinx/fem/dofmap.py

```diff
@@ -56,3 +66,22 @@ def index_map_bs(self):
     def list(self):
         """Adjacency list with dof indices for each cell."""
         return self._cpp_object.map()
+
+
+def create_dofmaps(
+    comm: Comm, topology: "dolfinx.mesh.Topology", elements: Sequence[FiniteElement]
+) -> list[DofMap]:
+    """Create degree-of-freedom maps on a given topology .
+
+    Args:
+        comm: MPI communicator
+        topology: Mesh topology
+        elements: Sequence of elements
+
+    Returns:
+        List of degree-of-freedom maps where the `i`-th map is the map for
```

Comment:
Doc strings use rst rather than Markdown. Need double, not single, ticks for fixed-width font.